src/newlines.cpp

```diff
@@ -3453,6 +3453,28 @@ static void newline_oc_msg(Chunk *start)
    {
       return;
    }
+   // Get count of parameters
+   size_t parameter_count = 0;
+
+   for (Chunk *pc = start->GetNextNcNnl(); pc->IsNotNullChunk(); pc = pc->GetNextNcNnl())
+   {
+      if (pc->level <= start->level)
+      {
+         break;
+      }
+
+      if (pc->Is(CT_OC_MSG_NAME))
+      {
+         parameter_count++;
+      }
+   }
+
+   int min_params = options::nl_oc_msg_args_min_params();
+
+   if (parameter_count < min_params && min_params != 0)
```

Comment:
I didn't spot this yesterday. As you can see in the rest of the code, we have a specific style for indenting multiple conditional expressions. ```&&``` will go to a new line and some extra spaces are needed in the first line. You can refer to any if or while statement with multiple conditional expressions in it for an example.

Reply:
Oh nice, just updated. Maybe we should add an uncrustify rule for that haha